pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -93,6 +93,10 @@ public PinotMetricsRegistry getMetricsRegistry() {
 
   public interface QueryPhase {
     String getQueryPhaseName();
+
+    default String getDescription() {
+      return "";
+    }
```

Comment:
why do we need the default behavior if the constructor is already catching and setting `""` ? ( are we concern backward-compatibility for other extensions?)